src/main/java/org/tron/program/SolidityNode.java

```diff
@@ -23,157 +24,221 @@
 import org.tron.core.config.DefaultConfig;
 import org.tron.core.config.args.Args;
 import org.tron.core.db.Manager;
-import org.tron.core.exception.AccountResourceInsufficientException;
-import org.tron.core.exception.BadBlockException;
-import org.tron.core.exception.BadItemException;
-import org.tron.core.exception.BadNumberBlockException;
-import org.tron.core.exception.ContractExeException;
-import org.tron.core.exception.ContractValidateException;
-import org.tron.core.exception.DupTransactionException;
-import org.tron.core.exception.NonCommonBlockException;
-import org.tron.core.exception.ReceiptCheckErrException;
-import org.tron.core.exception.TaposException;
-import org.tron.core.exception.TooBigTransactionException;
-import org.tron.core.exception.TooBigTransactionResultException;
-import org.tron.core.exception.TransactionExpirationException;
-import org.tron.core.exception.UnLinkedBlockException;
-import org.tron.core.exception.VMIllegalException;
-import org.tron.core.exception.ValidateScheduleException;
-import org.tron.core.exception.ValidateSignatureException;
 import org.tron.core.services.RpcApiService;
 import org.tron.core.services.http.solidity.SolidityNodeHttpApiService;
 import org.tron.protos.Protocol.Block;
-import org.tron.protos.Protocol.DynamicProperties;
 
 @Slf4j
 public class SolidityNode {
 
-  private DatabaseGrpcClient databaseGrpcClient;
   private Manager dbManager;
 
-  private ScheduledExecutorService syncExecutor = Executors.newSingleThreadScheduledExecutor();
+  private DatabaseGrpcClient databaseGrpcClient;
+
+  private AtomicLong ID = new AtomicLong();
+
+  private Map<Long, Block> blockMap = Maps.newConcurrentMap();
+
+  private LinkedBlockingDeque<Block> blockQueue = new LinkedBlockingDeque(10000);
+
+  private LinkedBlockingDeque<Block> blockBakQueue = new LinkedBlockingDeque(10000);

```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Explicitly declare the visibility for "blockBakQueue". ([squid:S2039](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2039))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1570&issues=AWYfmDct_XXc3YMrl43R&open=AWYfmDct_XXc3YMrl43R)

[ISSUE_KEY]: AWYfmDct_XXc3YMrl43R